test/Reviews.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Reviews from '../src/components/Reviews';
+
```

Comment:
import renderer from 'react-test-renderer'

This should get your props to work (it did for me, at least)